source/server/module.coffee

```diff
@@ -14,6 +13,9 @@ class Space.eventSourcing extends Space.Module
       snapshotting: {
         enabled: ['SPACE_ES_SNAPSHOTTING_ENABLED', true, 'bool']
         frequency: ['SPACE_ES_SNAPSHOTTING_FREQUENCY', 10, 'int']
+      },
+      mongo: {
```

Comment:
Yes this is good to clarify now. My original intent was make it forward compatible with being able to separate commits from snapshots, but that's early optimisation